README.md

```diff
@@ -1 +1,207 @@
+    },
+    {
+      "id": 7276925,
+      "node_id": "MDg6V29ya2Zsb3c3Mjc2OTI1",
+      "name": "ansible",
+      "path": ".github/workflows/ansible.yml",
+      "state": "active",
+      "created_at": "2021-03-29T16:04:44.000Z",
+      "updated_at": "2021-03-29T16:04:44.000Z",
+      "url": "https://api.github.com/repos/ansible-community/ppa/actions/workflows/7276925",
+      "html_url": "https://github.com/ansible-community/ppa/blob/main/.github/workflows/ansible.yml",
+      "badge_url": "https://github.com/ansible-community/ppa/workflows/ansible/badge.svg"
+    },
+    {
+      "id": 8751539,
+      "node_id": "MDg6V29ya2Zsb3c4NzUxNTM5",
+      "name": "resolvelib",
+      "path": ".github/workflows/resolvelib.yml",
+      "state": "active",
+      "created_at": "2021-05-04T03:28:11.000Z",
+      "updated_at": "2021-05-04T03:28:11.000Z",
+      "url": "https://api.github.com/repos/ansible-community/ppa/actions/workflows/8751539",
+      "html_url": "https://github.com/ansible-community/ppa/blob/main/.github/workflows/resolvelib.yml",
+      "badge_url": "https://github.com/ansible-community/ppa/workflows/resolvelib/badge.svg"
+    },
+    {
+      "id": 8751740,
+      "node_id": "MDg6V29ya2Zsb3c4NzUxNzQw",
+      "name": "ansible-core",
+      "path": ".github/workflows/ansible-core.yml",
+      "state": "active",
+      "created_at": "2021-05-04T03:46:52.000Z",
+      "updated_at": "2021-05-04T03:46:52.000Z",
+      "url": "https://api.github.com/repos/ansible-community/ppa/actions/workflows/8751740",
+      "html_url": "https://github.com/ansible-community/ppa/blob/main/.github/workflows/ansible-core.yml",
+      "badge_url": "https://github.com/ansible-community/ppa/workflows/ansible-core/badge.svg"
+    },
+    {
+      "id": 13724938,
+      "node_id": "W_kwDOFNAryM4A0W0K",
+      "name": "latest builds",
+      "path": ".github/workflows/latest_builds.yml",
+      "state": "active",
+      "created_at": "2021-09-30T21:15:42.000Z",
+      "updated_at": "2021-12-22T19:23:39.000Z",
+      "url": "https://api.github.com/repos/ansible-community/ppa/actions/workflows/13724938",
+      "html_url": "https://github.com/ansible-community/ppa/blob/main/.github/workflows/latest_builds.yml",
+      "badge_url": "https://github.com/ansible-community/ppa/workflows/latest%20builds/badge.svg"
+    }
+  ]
+}
+
+# Then (for example) to generate a new 5.3.0-2ppa~impish build for the ansible-5 PPA, you could make the following POST request. Note the 7276925 workflow ID used from the previous request to denote the ansible (.github/workflows/ansible.yml) workflow.
+
+% curl --location --request POST 'https://api.github.com/repos/ansible-community/ppa/actions/workflows/7276925/dispatches' \
+--header 'Authorization: token ghp_fazDzoWCbwbD9uf9PSMgKQUc9eDC9xp7bk2F' \
+--header 'Content-Type: application/json' \
+--data-raw '    {
+        "ref": "ansible-5",
+        "inputs": {
+            "DEB_DIST": "impish",
```

Comment:
Do we have to make multiple calls in case of different dists (impish, hirsute, focal, jammy)?

Reply:
it works with one or a space separated list, I've updated the example to include focal and impish